README.md

```diff
@@ -77,6 +88,9 @@ module load intel/compilers/2018.3
 module load intel/impi/2018.3/intel
 module load hdf5-intel/1.10.4
 ```
+Note that you should also check using ```bash module list``` that no competing
```

Comment:
This is just to fix the formatting of the Markdown code block.

````suggestion
Note that you should also check using 
```bash 
module list
```
that no conflicting
````